lib/MongoDB/Role/_CommandMonitoring.pm

```diff
@@ -269,4 +270,17 @@ sub _to_tied_ixhash {
     return \%out;
 }
 
+sub _maybe_expand_sections {
```

Comment:
Per my comment earlier, if you don't modify the query, then this shouldn't be needed.